src/components/carousel/index.jsx

```diff
@@ -37,10 +37,15 @@ const Carousel = ({
 	nextButton,
 	previousButton,
 	slidesToShow,
+	fullScreenShowButton,
+	fullScreenMinimizeButton,
+	enableFullScreenToggleButton,
```

Comment:
Not sold on my thoughts here, but `enableFullScreenToggleButton` feels verbose.  I think I would prefer to see `enableFullScreen` since it's enabling the functionality, not just the button.

Reply:
gotcha. yeah I can change it if that feels weird. I mostly didn't use `enableFullScreen` because I didn't want people to think that that was going to be full-screen on first render if it was enabled

Reply:
`enableFullScreenOption` works.

I think if the variable was `fullScreen` that might indicate that it's full screen at start, but I can settle on the `enableFullScreenOption`.  I'm just thinking about it as a tag option like so:
```
  <Carousel fullScreen /> /* this feels very much like fullScreen is active */
  <Carousel enableFullScreen />  /* this feels like it's enabling the fullScreen feature to me */
  <Carousel enableFullScreenOption /> /* this feels alright to me though, I'm ok with this little verbosity */
  <Carousel enableFullScreenToggleButton /> /* this just feels like it's enabling the button, as odd as that sounds */
```

Reply:
it's ok. @nschubach I went with your original suggestion! thought about it some more and it's cool